virt-v2v/cold/BUILD.bazel

```diff
@@ -89,15 +90,29 @@ container_image(
     visibility = ["//visibility:public"],
 )
 
+go_library(
+    name = "cold_lib",
+    srcs = ["entrypoint.go"],
+    importpath = "github.com/konveyor/forklift-controller/virt-v2v/cold",
+    visibility = ["//visibility:private"],
+)
+
+go_binary(
+    name = "cold",
+    embed = [":cold_lib"],
+    visibility = ["//visibility:public"],
+)
+
 container_image(
     name = "forklift-virt-v2v",
     base = ":virt-v2v-layer",
     directory = "/usr/local/bin/",
     empty_dirs = ["/disks"],
-    entrypoint = ["/usr/local/bin/entrypoint"],
-    env = {"LIBGUESTFS_BACKEND": "direct"},
+    entrypoint = ["/usr/local/bin/cold"],
```

Comment:
i didn't line it's being called `cold`, here in the file, but that's only a matter of taste

Reply:
changed.